physics/smoke_dust/dep_data_mod.F90

```diff
@@ -0,0 +1,193 @@
+!>\file  dep_data_mod.F90
+!! This file contains data for the dry deposition modules.
+module dep_data_mod
+
+  use machine , only : kind_phys
+
+  integer, parameter :: nvegtype = 25
+  real(kind_phys), dimension(nvegtype), parameter :: &
+        kpart = (/500., 500., 500., 500., 500., &
+                  500., 500., 500., 500., 500., &
+                  500., 500., 500., 500., 500., &
+                  500., 500., 500., 500., 500., &
+                  500., 500., 500., 500., 500.    /)
+  real(kind_phys), parameter :: max_dep_vel = 0.005                   ! m/s (may need to set per species)
+  real(kind_phys), parameter :: dep_ref_hgt = 2.0                     ! Meters 
+  real(kind_phys), parameter :: pi = 3.1415926536
```

Comment:
I'll reiterate @dustinswales comment about redefining physical constants and such in this module. We would like to see removal of things that already have been declared and have values from the host. This reduces inconsistencies in the physics. Again, I'm not requiring this to change at this late stage of testing, just pointing out that it should be changed in the future. We should be able to help with this.